src/main/java/frc/robot/scoreStates/ClimbState.java

```diff
@@ -33,17 +33,17 @@ public Command driveToClosestOptimalPoint() {
     }
 
     public Command initializeCommand() {
-        return elevator.setHeight(ElevatorConstants.STARTING_CLIMB_HEIGHT);
+        return Commands.none();
     }
```

Comment:
It might be better to just give these functions default implementations in the interface.

Reply:
16a32ce94ba75136a6bb7c55ac6807c425f8158e